### PR TITLE
Rearranging site to use Managed JupyterHubs and Projects

### PR DESCRIPTION
**Pull request by choldgraf**
This covers much of the to-do items in https://github.com/2i2c-org/team-compass/issues/62

It re-works the content and structure according to the outline there. It is mostly a rearranging rather than creating new content, though there are some new sections added (such as information about major projects we currently collaborate on...@rabernat and @fperez let me know if the way we describe our relationship with Pangeo/JMTE should be corrected!)

Would also love thoughts from @yuvipanda on this!

Demo here: https://deploy-preview-53--cocky-booth-e7ed17.netlify.app/

### CODE REVIEW

**Comment by yuvipanda** [comment]
Extra underline in the button in 'projects' page:

image

Same on the managed JupyterHubs page. I definitely like the re-org! We should do another round on the copy for brevity, but otherwise +1.

**Comment by choldgraf** [comment]
ok - I just took another pass and removed some text + streamlined, and also fixed the underline CSS thing you highlighted 👍

File: content/home/mission.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We make interactive computing more accessible and powerful for research and educ
 <div class="card-group mission-cards">
     <div class="card w-50 m-3" style="width: 18rem;">
         <div class="card-body text-center">
-            <h5 class="card-title">Manage interactive computing infrastructure for research and education</h5>
+            <h5 class="card-title">Manage and build interactive computing infrastructure for research and education</h5>
             <p class="card-text">
             2i2c provides managed JupyterHubs for data science in research and engineering communities. They are tailored for the communities they serve and 💯 open source.
```
Comment (fperez) [inline content/home/mission.md]: Not part of the diff, but I'd suggest using "offers" instead of "provides" here. The idea is that we offer something that others must decide to accept and purchase. To me, "provides" sounds like something that's always available, whereas I think our offerings have more of a "product" flavor to them...

Ignore if it's my non-native English brain misreading though...

**Comment by fperez** [comment]
This looks great, thanks! Left a minor note, otherwise big +1 from me.

File: content/values/open-source.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+weight = 10
+title = "Open source tools for research and education"
+
+[design]
+  columns = "1"
++++
+
+2i2c supports and uses projects that are developed, controlled, and owned by the community. We believe in the importance of tools that are built by a diverse community of multiple stakeholders, in order to build tools that are powerful for all of us, and in an equitable and inclusive manner.
+
+Open source tools exist across all parts of the data science stack - from the domain-specific libraries you use to analyze data, to the interfaces that allow you to interact with the computer, to the software that orchestrates your workflow in the cloud. 2i2c believes it is crucial that research and education has access to open source tools across **all layers of this stack**.
```
Comment (GeorgianaElena) [inline content/values/open-source.md]: I love this! <3

The only thing that I'd rephrase is the `has access` in this sentence:
> 2i2c believes it is crucial that research and education has access to open source tools across **all layers of this stack**.

In my mind, everyone has access to all the open source tools, by definition <3, and what 2i2c does, is rather to put these together and give access to the end machinery.

(Hope I made some sense and this didn't come out as an undecipherable idea :sweat_smile:)

File: content/projects/active/pangeo.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+weight = 41
+title = "Pangeo Hub Infrastructure"
+
+[design]
+  columns = "2"
++++
+
+{{< figure src="https://pangeo.io/_images/pangeo_simple_logo.svg" class="projects-image" >}}
+
+The [Pangeo project](https://pangeo.io/) is a community platform for Big Data geoscience. 2i2c collaborates with Pangeo by developing and running JupyterHub infrastructure for the Pangeo Hubs. This work focuses around building collaborative data science platforms that can draw from large cloud datasets, as well as integrating JupyterHub with scalable computing in the cloud via [Dask Gateway](https://gateway.dask.org/).
```
Comment (rabernat) [inline content/projects/active/pangeo.md:14]: :+1:

**Review by rabernat — APPROVED** [review]
Great work Chris!

**Comment by choldgraf** [comment]
OK I'm seeing lots of 👍 and I slightly modified the language per @GeorgianaElena's suggestion - I'm gonna merge this and we can iterate in the future!
